spyder/utils/introspection/manager.py

```diff
@@ -154,7 +183,18 @@ def _handle_timeout(self):
         self.waiting = False
         if self.pending:
             self._finalize(self.pending)
+        elif self.pending_request:
+            info = self.pending_request
+            self.pending_request = None
+            self.send_request(info)
         else:
+            self.timeout_counter += 1
+            if (self.timeout_counter > MAX_TIMEOUTS and
+                    not self.plugins_initialized['jedi']):

```

Comment:
1. Yep, that is the way.
2. I see that when Jedi plugin is not started, although the Rope plugin and the fallback plugin are up, no content is given for completions for libraries like `numpy`